### ISSUE TITLE
srs does not respect http status

### ISSUE THREAD

**Issue by akryl9296** [winlinvip/simple-rtmp-server#158]
on_publish srs doesn't respect appropriate http status - has received status 403 and integer 0 in plain text, and proceeded to streaming anyway. Documentation says status 200 is expected AND integer 0 in plain text https://github.com/winlinvip/simple-rtmp-server/blob/master/trunk/conf/full.conf#L357

**Comment by winlinvip** [comment]
I will fix it in 2.0

**Comment by winlinvip** [comment]
on_connect response 401:
```
[2015-01-02 10:19:34.215][error][29192][109][11] http hook on_connect status failed. client_id=109, code=401, ret=3049(Resource temporarily unavailable)
[2015-01-02 10:19:34.215][error][29192][109][9] hook client on_connect failed. url=http://127.0.0.1:8085/api/v1/clients, ret=3049(Bad file descriptor)
[2015-01-02 10:19:34.215][error][29192][109][9] check vhost failed. ret=3049(Bad file descriptor)
```
